### PR TITLE
Mobile v2.12 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/deploy/mobile-app-changelog.md [hunk 1]
```diff
@@ -1,13 +1,44 @@
...
+ - Calls: Incoming call notifications now appear on the channel list screen.
+ - Calls: Added the server name to the call notification banner for a better user experience.
+ - Calls: Added support for multiple client sessions for the same user in the same call.
+ - Added a new option to the **Channel Info** to be able to auto-follow all threads.
+ - Added an option to convert Group Messages to private channels.
+ - Added a ``channelType`` parameter to ``persistentNotificationsConfirmation``.
+ - Added a new feature to highlight keywords without triggering a notification. This cannot be configured in the mobile app.
+
+### Bug Fixes
+ - Calls: Fixed an issue where the in-app "push" notification overlay appeared when the user already received the incoming call notification banner.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:28]:
```suggestion
 - Calls: Fixed an issue where the in-app "push" notification overlay appeared after the user had already received the incoming call notification banner.
```

File: source/deploy/mobile-app-changelog.md [hunk 2]
```diff
@@ -1,13 +1,44 @@
...
+ - Fixed an issue where mentions did not appear in the global **Thread** view.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:29]:
```suggestion
 - Fixed an issue where mentions did not appear in the global **Threads** view.
```

File: source/deploy/mobile-app-changelog.md [hunk 3]
```diff
@@ -1,13 +1,44 @@
...
+ - Fixed an issue where an error "Cannot read property 'centerChannelColor' of undefined" appeared.
+ - Fixed an issue with opening the same modal twice.
+ - Fixed "Invariant Violation: "Home" has not been registered" error when opening the app.
+ - Fixed an issue where toggling of emojis in the emoji picker and quick reactions was not enabled.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:33]:
```suggestion
 - Fixed an issue where toggling emojis in the emoji picker and quick reactions were not enabled.
```

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline.

**Review by cpoile — APPROVED** [review]

**Review by marianunez — APPROVED** [review]

**Review by enahum — APPROVED** [review]

**Review by larkox — APPROVED** [review]

**Review by johndavidlugtu — APPROVED** [review]

**Review by streamer45 — APPROVED** [review]

**Pull request merged**